LoopKit/Service/VersionCheckService.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  VersionCheckService.swift
+//  LoopKit
+//
+//  Created by Rick Pasetto on 9/8/21.
+//  Copyright © 2021 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+public protocol VersionCheckService: Service {
+
+    func checkVersion(currentVersion: String, completion: @escaping (Result<VersionUpdate, Error>) -> Void)
```

Comment:
Sure thing.  I will note that the `/info` API call on TidepoolService does not take a parameter...I guess in theory it could take `bundleIdentifier` and respond in kind, too?  Just a thought...

Reply:
Right, but that's the difference between the `VersionCheckService` (a generic service used to check the version requirements for a particular app) and the `TidepoolService` implementation which is returning version requirements for the *specific* Loop app (or potentially other apps reported by the Tidepool backend).

Reply:
Sorry, maybe my point was missed.  The `/info` API call, which `TidepoolService` uses (via `TAPI`) returns a struct with versions for not just Loop, but other mobile apps as well.  We could pass along the `bundleIdentifier` so we don't have to just keep adding payload to that response every time a new app ships.